.travis.yml

```diff
@@ -70,7 +70,7 @@ script:
   - if [[ $CHECK_CS == 'true' ]]; then composer cs-check ; fi
 
 after_script:
-  - if [[ $TEST_COVERAGE == 'true' ]]; then travis_retry composer upload-coverage ; fi
+  - if [[ $TEST_COVERAGE == 'true' ]]; then travis_retry php vendor/bin/php-coveralls -v ; fi
```

Comment:
Can't we have just:
` - if [[ $TEST_COVERAGE == 'true' ]]; then vendor/bin/php-coveralls -v ; fi`
as https://github.com/zendframework/maintainers/blob/master/template/.travis.yml#L52

Reply:
Ok, fine, I'll update then maintainers repo with these changes.